src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1191,10 +1167,10 @@ export class CreateVmAction extends React.Component {
                     vms={this.props.vms}
                     osInfoList={this.props.systemInfo.osInfoList}
                     onAddErrorNotification={this.props.onAddErrorNotification}
-                    cloudInitSupported={this.state.cloudInitSupported}
-                    downloadOSSupported={this.state.downloadOSSupported}
-                    unattendedSupported={this.state.unattendedSupported}
-                    unattendedUserLogin={this.state.unattendedUserLogin}
+                    cloudInitSupported={this.props.virtInstallFeatures.cloudInitSupported}
```

Comment:
The idea was that this could just be `virtInstallFeatures={this.props.virtInstallFeatures}`, otherwise wrapping the individual props into a structure is no win at all, no?